<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>499</width>
    <height>354</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sign In demoEcommerce.db</string>
  </property>
  <widget class="QPushButton" name="pushButtonDelete">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>150</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete User</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelResponse">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>260</y>
     <width>371</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>250</y>
     <width>391</width>
     <height>80</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>51</x>
     <y>51</y>
     <width>391</width>
     <height>91</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Email Address</string>
      </property>
      <property name="buddy">
       <cstring>lineEditEmailAddress</cstring>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Password</string>
      </property>
      <property name="buddy">
       <cstring>lineEditPassword</cstring>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEditEmailAddress"/>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEditPassword">
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="labelSure">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>181</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
   <property name="buddy">
    <cstring>lineEditPassword</cstring>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonYes">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>220</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Yes</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonNo">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>220</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>No</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEditEmailAddress</tabstop>
  <tabstop>lineEditPassword</tabstop>
  <tabstop>pushButtonDelete</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonDelete</sender>
   <signal>pressed()</signal>
   <receiver>Dialog</receiver>
   <slot>DeleteUser()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>282</x>
     <y>181</y>
    </hint>
    <hint type="destinationlabel">
     <x>425</x>
     <y>178</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonYes</sender>
   <signal>pressed()</signal>
   <receiver>Dialog</receiver>
   <slot>ConfirmDelete()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>169</x>
     <y>216</y>
    </hint>
    <hint type="destinationlabel">
     <x>39</x>
     <y>165</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonNo</sender>
   <signal>pressed()</signal>
   <receiver>Dialog</receiver>
   <slot>NoConfirmDelete()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>343</x>
     <y>222</y>
    </hint>
    <hint type="destinationlabel">
     <x>435</x>
     <y>219</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>DeleteUser()</slot>
  <slot>ConfirmDelete()</slot>
  <slot>NoConfirmDelete()</slot>
 </slots>
</ui>
